### DISCUSSION TITLE
Need help : Split string

### DISCUSSION THREAD

**Discussion by Yeonju09** [ClickHouse/ClickHouse#62465] [Q&A]
Hi!

I'm not from a technical background and I'm new to GitHub as well as Clickhouse. Please bear with me!

I have a column called 'event_url' and it has various URLs.
Some URLs look like this : **https://www.domain_name.com/event/*******
I was wondering if there's a way to create a column that has strings after **'/event/'**.
If there's no '/event/' in the url then it can output null.

It would be great if anyone could give me a clickhouse SQL query to do this. Please let me know if you need any more information. Thank you so much!!

**Comment by HowePa** [comment]
You mean this?

```SQL
SELECT url LIKE '%event%' ? url : 'null' FROM {table}
```

![image](https://github.com/ClickHouse/ClickHouse/assets/35292408/15888e07-8561-4ab9-80cc-917f90fb13ff)

or just

```SQL
SELECT url FROM {table} WHERE url LIKE '%event%'
```
![image](https://github.com/ClickHouse/ClickHouse/assets/35292408/8522fcdb-86ac-4654-9a06-7b23fecc6551)
